iroh/src/node.rs

```diff
@@ -7,18 +7,35 @@
 //! To shut down the node, call [`Node::shutdown`].
 use std::fmt::Debug;
 use std::future::Future;
+use std::io;
```

Comment:
Yes, I was going to do this. Already discussed with @Frando how to make a rpc request protocol more modular and split up the handlers.

The question is if this should happen in this PR or a subsequent one. I would prefer doing it later in a separate PR, otherwise _somebody_ will be in merge hell.